picture-in-picture/index.js

```diff
@@ -31,3 +29,18 @@ function onPipWindowResize(event) {
   const pipWindow = event.target;
   log(`> Window size is ${pipWindow.width}x${pipWindow.height}`);
 }
+
+
+// Hide button if Picture-in-Picture is not supported.
+togglePipButton.hidden = !('pictureInPictureEnabled' in document);
+
+// Set button ability if Picture-in-Picture can be used.
+setPipButton();
+
+video.addEventListener('loadedmetadata', setPipButton);
+video.addEventListener('emptied', setPipButton);
+
+function setPipButton() {
+  togglePipButton.disabled = true ? (video.readyState === 0) :
+      !document.pictureInPictureEnabled || video.disablePictureInPicture;
```

Comment:
`true ?` means you will only check for the `readyState` or am I missing something?

Reply:
I was thinking about checking `readyState === 0` only but this ternary thing makes it harder to read. So I've removed it.

Thanks!